packages/utils/pack-up/src/__tests__/cli.test.ts

```diff
@@ -9,9 +9,9 @@ describe('cli', () => {
     await cleanupWorkspaces();
   });
 
-  const timeout = 1000 * 50;
+  const timeout = 1000 * 120;
 
-  describe.skip('build & check', () => {
+  describe('build & check', () => {
```

Comment:
@innerdvations 😘

Reply:
❤️ but also `timeout = 1000 * 120` 😳 I know it's necessary but our poor CI is already struggling. Hopefully it's considerably faster and that's just a safety mechanism.